### PR TITLE
Unit test for ensureFollowingPeergos

### PR DESCRIPTION
**Pull request by cesarosum**

### CODE REVIEW

File: src/peergos/server/tests/UserTests.java [hunk 1]
```diff
@@ -264,6 +264,23 @@ public void changeLoginAlgorithm() throws Exception {
...
+        String username = generateUsername();
+        String password = "password";
+        UserContext userContext = PeergosNetworkUtils.ensureSignedUp(username, password, network, crypto);
+        Boolean isFollowingPeergos = userContext.getFollowing().get().contains("peergos");
+        assertTrue("Following Peergos after sign-up", isFollowingPeergos.equals(true));
+        userContext.unfollow("peergos");
+        isFollowingPeergos = userContext.getFollowing().get().contains("peergos");
+        assertTrue("Unfollowed Peergos", isFollowingPeergos.equals(false));
+        userContext.logout();
+        userContext.signIn(username, password, network, crypto);
```
[the remark this answers is not in the document]
Comment (cesarosum) [reply src/peergos/server/tests/UserTests.java]: OK this has been replaced now.

**Comment by cesarosum** [comment]
@ianopolous OK this is looking much better now.

File: src/peergos/server/tests/UserTests.java [hunk 2]
```diff
@@ -264,6 +264,23 @@ public void changeLoginAlgorithm() throws Exception {
...
 
+    @Test
+    public void ensureUserIsTryingToFollowPeergos() {
+        String username = generateUsername();
+        String password = "password";
+        UserContext userContext = PeergosNetworkUtils.ensureSignedUp(username, password, network, crypto);
+        Boolean requestToFollowPeergos = userContext.getSocialState().join().pendingOutgoingFollowRequests.containsKey("peergos");
+        assertTrue("Pending follow request Peergos exists after sign-up", requestToFollowPeergos);
+        userContext.unfollow("peergos");
+        requestToFollowPeergos = userContext.getFollowing().join().contains("peergos");
```
Comment (ianopolous) [inline src/peergos/server/tests/UserTests.java]: This will never be true without interaction from the peergos user

Comment (ianopolous) [reply src/peergos/server/tests/UserTests.java]: "I would log in as peergos and accept the request, then get the user to remove peergos as a follower

**Pull request closed without merging**